charts/prometheus-postgres-exporter/templates/deployment.yaml

```diff
@@ -37,6 +37,12 @@ spec:
 {{ toYaml .Values.annotations | indent 8 }}
 {{- end }}
     spec:
+      nodeSelector:
+{{ toYaml (default .Values.global.platformNodePool.nodeSelector .Values.nodeSelector) | indent 8 }}
```

Comment:
IMHO we should nindent all of these so they are on a single line. it makes reading the template easier.